django/cantusdb_project/main_app/admin.py

```diff
@@ -22,7 +22,11 @@ class CenturyAdmin(BaseModelAdmin):
     pass
 
 
+@admin.register(Chant)
```

Comment:
Is there a reason we're registering this area of the admin differently than the others?

Reply:
This was a change that Andrew made, I don't think there is any real difference so we can change it back if you think that's best.

Reply:
I just want things to be consistent, unless there's a good reason for the inconsistency!